src/unix/linux_like/android/b64/x86_64/mod.rs

```diff
@@ -4,6 +4,12 @@ pub type greg_t = i64;
 pub type __u64 = ::c_ulonglong;
 
 s! {
+    pub struct ip_mreqn {
+        pub imr_multiaddr: ::in_addr,
+        pub imr_address: ::in_addr,
+        pub imr_ifindex: ::c_int,
+    }
```

Comment:
This struct needs to be provided on all Android targets, not just x86_64. Otherwise the CI checks on merge will fail (bors tests some extra targets).

Reply:
Done, thanks.